src/20_Components/amp-addthis.html

```diff
@@ -0,0 +1,84 @@
+<!---{
+  "disablePlayground": true
+}--->
+<!--
+  ## Introduction
+  AddThis Floating Bar and Inline Share Buttons are available for AMP! Make it simple for website visitors to share your content with our free social sharing buttons. Allow your visitors to connect directly to 200+ social networks, including Facebook, Twitter, WhatsApp, and more.
+
+  AddThis is trusted by over 15,000,000 websites with over 2 billion unique users, sharing content all over the world, in more than sixty languages.
+-->
+<!-- -->
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="/components/amp-addthis/">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>amp-addthis</title>
+  <script async custom-element="amp-youtube" src="https://cdn.ampproject.org/v0/amp-youtube-0.1.js"></script>
+  <!-- ## Setup -->
+  <!--
+  Import the `amp-addthis` component.
+  -->
+  <script async custom-element="amp-addthis" src="https://cdn.ampproject.org/v0/amp-addthis-0.1.js"></script>
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    @keyframes forcePositioning {
+      from,
+      to {
+        position: relative;
+      }
+    }
+
+    amp-addthis[data-widget-type=floating] {
+      animation: 1s linear infinite forcePositioning;
+    }
+  </style>
+</head>
+<body>
+  <!-- ## Share Button Types -->
```

Comment:
We're currently in the process of moving away from title case for headings. Can you replace this with: `Share button types`

Reply:
Happy to keep this on par with the rest of the site. An update will be coming shortly.